src/user/mod.rs

```diff
@@ -20,6 +20,8 @@ pub use self::middleware::{Middleware, RequestUser};
 
 pub mod middleware;
 
+/// The model representing a row in the `users` database table.
+///
```

Comment:
I felt like it was much easier to read the doc if they have a little room to breathe, but if this is not desired I can remove the extra line